src/Shared/Process/ProcessEx.cs

```diff
@@ -179,7 +179,7 @@ private void OnProcessExited(object sender, EventArgs e)
                 _output.WriteLine("Process exited.");
             }
         }
-
+        _process.WaitForExit();
```

Comment:
Should have a comment so we don't remove the line again

Reply:
Added one